src/main/java/core/basesyntax/dao/StorageDaoImpl.java

```diff
@@ -8,8 +8,11 @@ public class StorageDaoImpl implements StorageDao {
 
     @Override
     public User add(User user) {
-        user.setId(++index);
-        Storage.people.add(user);
+        if (user.getLogin() != null && user.getPassword() != null && user.getAge() != null
+                && user.getLogin().length() >= 6 && user.getPassword().length() >= 6) {
+            user.setId(++index);
+            Storage.people.add(user);
+        }
```

Comment:
don't change DAO
what if those conditions don't pass - we just silently ignore `add()`? That's a bad design. We validate data in service and throw exceptions to inform user what is wrong